tests/utils/import.ts

```
@@ -57,7 +57,19 @@ export function validateResponseSpec(
   )
 
   if (!validate) {
+    for (const error of ajv.errors) {
+      if (
+        error.hasOwnProperty("params") &&
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)